### DISCUSSION TITLE
Replace 🏰 BuidlGuidl with the BuidlGuidl logo

### DISCUSSION THREAD

**Discussion by ZakGriffith** [scaffold-eth/scaffold-eth-2#549] [Ideas]
Since we have clean logos now, it would be nice to update the footer BuidlGuidl wording from 🏰 BuidlGuidl to the actual logo.  We hav both light and dark modes available so that would need to be implemented as well.  The logo files can be found in this Figma:
https://www.figma.com/file/jPaXdmHNfbTSwBWwENr6Iq/BG-Media-Kit?type=design&node-id=0-1&mode=design&t=mvJiUwXPXhZ7LQaz-0

**Comment by FilipHarald** [comment]
I can give this a try. Should I create an issue then?

**Comment by FilipHarald** [comment]
If I understand the Contributing-docs correctly an issue is only needed for larger PRs. [I gave it a shot here](https://github.com/scaffold-eth/scaffold-eth-2/pull/551). I don't know if the CSS is optimal, but maybe I will get feedback on that in PR.

**Comment by carletex** [comment]
Fixed on https://github.com/scaffold-eth/scaffold-eth-2/pull/551
